src/main/java/com/fasterxml/jackson/databind/ser/std/SqlBlobSerializer.java

```diff
@@ -0,0 +1,56 @@
+package com.fasterxml.jackson.databind.ser.std;
+
+import java.io.IOException;
+import java.sql.Blob;
+import java.util.Base64;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.annotation.JacksonStdImpl;
+
+/**
+ * This is serializer for {@link java.sql.Blob}  into base64 String
+ */
+
+@JacksonStdImpl
+@SuppressWarnings("serial")
+public class SqlBlobSerializer
+extends JsonSerializer<Blob>
```

Comment:
Let's use `StdScalarSerializer` as base instead (serializes as scalar value, JSON String; as opposed to structured Object/Array).